cockroachdb/README.md

```diff
@@ -11,6 +11,10 @@ The CockroachDB check monitors the overall health and performance of a [Cockroac
 The CockroachDB check is included in the [Datadog Agent][2] package, so you do not
 need to install anything else on your server.
 
+Starting with version 1.9.0, this OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. Note that the latest mode requires Python 3. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][14].
```

Comment:
This is where it gets a bit tricky because not all integrations have used the exact same config options to enable one mode or the other. In this case, it is the presence of the `openmetrics_endpoint` that triggers the change: https://github.com/DataDog/integrations-core/blob/ed10099eb29d286a1e7859bbff16046007d29c8a/cockroachdb/datadog_checks/cockroachdb/cockroachdb.py#L18-L30

Let me see if I can find a short enough way to express that within our cookiecutter paragraph. Feel free to improve on it.

```suggestion
Starting with version 1.9.0, this OpenMetrics-based integration has a latest mode (enabled by setting `openmetrics_endpoint` to point to the target endpoint) and a legacy mode (enabled by setting `prometheus_url` instead). To get all the most up-to-date features, Datadog recommends enabling the latest mode. Note that the latest mode requires Python 3. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][14].
```